content/summit2025/confirmed-talks.html

```diff
@@ -0,0 +1,68 @@
+---
+url: /summit2025/talks
+layout: index
+title: "Transparency.dev Summit 2025 Confirmed Talks"
+description: "Talks that have been confirmed for the Transparency.dev Summit 2025"
+---
+<section class="glue-page gmt-6 gmb-7">
+  <h1 class="headline2 sm:text-center gmb-4">
+    <div class="eyebrow">Transparency.dev Summit</div>
+    Confirmed Talks
+  </h1>
+  <div class="glue-grid gmt-4 gmb-2">
+    <div class="col-span-4 sm:col-span-8 sm:col-start-3">
+      <table class="w-full text-sm text-left rtl:text-right text-gray-500 dark:text-gray-400 gmb-2">
+        <thead class="text-xs text-gray-700 uppercase dark:text-gray-400">
+          <tr>
+            <th scope="col" colspan="3" class="px-6 py-3 bg-gray-50 dark:bg-gray-800">
+              Accepted Talks
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr class="border-b border-gray-200 dark:border-gray-700">
+            <td class="px-6 py-3">
+              Brendan McMillion
+            </td>
+            <td class="px-6 py-3">
+              <a href="/summit2025/talks/kt-brendon.html">Key Transparency / When Collusion-Based Security Fails</a>
+            </td>
+            <td class="px-6 py-3">
+              Lightning talk
+            </td>
```

Comment:
@rgdd and I decided to leave off lightning vs full for the time being. Do you think it's better to have it?

Reply:
I can remove it easily enough.  On the pages with the talk details it says when a talk is a lightning talk too, I'll make them all look like standard talks for now?

Reply:
Done, PTAL